src/lib/web-inspector.js

```diff
@@ -195,6 +202,33 @@ WebInspector.ConsoleMessage.MessageType = {
   Log: 'log'
 };
 
+// Mock NetworkLog
+WebInspector.NetworkLog = function(target) {
+  this._requests = [];
+  target.networkManager.addEventListener(
+    WebInspector.NetworkManager.EventTypes.RequestStarted, this._onRequestStarted, this);
+};
+
+WebInspector.NetworkLog.prototype = {
+  requests: function() {
+    return this._requests;
+  },
+
+  requestForURL: function(url) {
+    for (var i = 0; i < this._requests.length; ++i) {
```

Comment:
I guess we'll never have a large number of requests, but would it be better to just have an object (or Map) here, keyed on the url?